include/transformations/RenameEntitiesTransformation.h

```diff
@@ -30,14 +30,16 @@ std::discrete_distribution;
 /// RecursiveASTVisitor is a set of actions that are done
 /// when a certain node of AST is reached. RenameEntitiesVisitor in particular
 /// visits all entities (e.g functions or variables) and rename them to a
-/// random name
+/// random name (or name like prefix_(hash from old name)
 class RenameEntitiesVisitor : public RecursiveASTVisitor<RenameEntitiesVisitor> {
  public:
     explicit RenameEntitiesVisitor(Rewriter * rewriter, const bool rename_func, const bool rename_var,
                                    discrete_distribution<int> token_len_generator,
                                    discrete_distribution<int> tokens_num_generator,
                                    discrete_distribution<int> char_generator,
-                                   mt19937 * gen, const bool test = false);
+                                   mt19937 * gen, string  hash_prefix = "d",
```

Comment:
Also, I think it is better to pass a pointer to `hash_prefix` to avoid copying. I remember your suggestion about moving from pointers to references, but let's use the pointer in the current pull request